src/TodoListItem.tsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
-interface TodoListItemProps {
-    item: string;
-}
-const TodoListItem: React.FC<TodoListItemProps> = ({item}) => {
+import {TodoListItemProps} from "./types.ts";
+
+const TodoListItem: React.FC<TodoListItemProps> = ({item, removeTodo}) => {
     return (
-        <li>{item}</li>
+        <li>{item.title}&nbsp;<button onClick={()=>removeTodo(item.id)}>Remove</button></li>
```

Comment:
Super minor nit: there's some talk about when to use `&nbsp;` and when not to do so. Seems the guidance is to use it to separate words specifically. For cases like this one, CSS is recommended, since nbsp may behave differently depending on browser/OS. Definitely feel free to challenge me